test/test_type_promotion.py

```diff
@@ -1165,6 +1165,19 @@ def expected_type(inp, val):
                     actual = torch.clamp_max_(inp, val)
                     self.assertEqual(actual, expected, exact_dtype=False)
 
+    @onlyNativeDeviceTypes
+    def test_ternary_out_promotion(self, device):
```

Comment:
I would expect that test change for these would be done by adding more sample to the OpInfo?

Reply:
In general I agree, but unless I'm looking at the wrong location, I could not find any examples where we test type promotion for out. I can add `Case 5` to `test_out`, see
https://github.com/pytorch/pytorch/blob/82e428723aa5ae31569d2c98f92b420a03a22978/test/test_ops.py#L780-L785
But I'm a bit scared about numerous regressions it can find in different OPS and as such I propose to track it in a separate issue